roles/kube-template-cni/tasks/main.yml

```diff
@@ -22,7 +22,13 @@
     src: flannel-rbac.yaml.j2
     dest: /etc/flannel-rbac.yaml
 
-- name: Template bridge.yaml
+#- name: Template bridge.yaml
```

Comment:
I'm not a big fan of leaving commented code here. We can always refer to git history if we needed to. Remove if it's not really needed in this section.

Reply:
Changed (this should not commented, so I've reverted)